### DISCUSSION TITLE
23 gennaio 2023 A - esercizio 3

### DISCUSSION THREAD

**Discussion by atyion** [sapienzastudentsnetwork/basi-di-dati-1#22] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/9e64e8d4-194e-4441-9c65-f5a6da4fff45)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/234959a9-ee78-495f-a814-ff01951873c8)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/3f9c4cb7-2f28-4212-a6a5-e7a9e8da03bd)

**Reply by atyion** [reply]
ATTENZIONE: Ho sbagliato RpBk, è limite superiore invece di essere limite inferiore. Domani quando ho tempo vedo di pubblicare la versione corretta.

**Comment by niccolo-fato** [comment]
Soluzione analoga a @atyion: 
![IMAGE 2024-01-13 17:51:15](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/5c90b4c9-50fa-4e7c-a779-fc0b43429523)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/275b91ee-c1eb-4ee5-a588-30409d6b4d9c)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/0ae67e52-530b-4862-b436-78d6d4d58621)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/ce2e6bb8-f074-40d7-8afb-677895cbcfa7)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/17b6d033-14c8-44c5-a611-4782aa211718)
